ui/src/journals/containers/SearchPageContainer.tsx

```diff
@@ -15,12 +16,23 @@ import { isCataloger } from '../../common/authorization';
 const META_DESCRIPTION = 'Find journals publishing about High Energy Physics';
 const TITLE = 'Journals Search';
 
+export interface JournalMetadata {
+  short_title: string;
+  journal_title: string;
+  urls: string[];
+  control_number: string;
```

Comment:
is a string or a number? We are using it inconsistently for example [here](https://github.com/inspirehep/inspirehep/pull/2697/files#diff-b28d9d340beff2217c916f019c8f197385f9532ff78264067457d37b0ec7bc5bR31) we are expecting it as a number

Reply:
@karolina-siemieniuk-morawska ☝️

Reply:
Unified to type number (correct type) across files